test/unit/instance/save.test.js

```diff
@@ -11,11 +11,18 @@ const sinon     = require('sinon');
 
 describe(Support.getTestDialectTeaser('Instance'), () => {
   describe('save', () => {
-    it('should disallow saves if no primary key values is present', async () => {
+    it('is not allowed if the instance does not have a primary key defined', async () => {
+      const User = current.define('User', {});
+      const instance = User.build({});
+
+      await expect(instance.increment()).to.be.rejectedWith('but this model instance is missing the value of its primary key');
```

Comment:
```suggestion
      await expect(instance.save()).to.be.rejectedWith('but this model instance is missing the value of its primary key');
```

Reply:
Damn I don't even know how I missed these, thanks a lot for the patience!